scripts/build/commands/test.js

```diff
@@ -0,0 +1,42 @@
+const path = require('path');
+
+const fse = require('fs-extra');
+const rimraf = require('rimraf');
+
+const {exec} = require('../../utils/exec');
+
+module.exports = {
+  command: 'test <distPath>',
+  desc: 'Bundle the main widgets into a distributable folder for testing',
```

Comment:
Not a fan of the name `test` for this command. Maybe `test-bundle` or something else?

Reply:
👍 That makes sense